src/nrfDevice.ts

```diff
@@ -103,6 +96,22 @@ export const nrfdevice = (
             );
             deviceAssociated = true;
 
+            //Init shadow
+            console.log(
+              'Shadow',
+              res?.data?.state?.reported?.simulator,
+              res?.data?.state,
+            );
+            if (
+              (appType ||
+                (!res?.data?.state?.reported?.simulator && !appType)) &&
+              !shadowInitialized
```

Comment:
When will there be no `appType`? If they give no `-t` option? But then won't `appType` be `atv2`? (I didn't trace this down)

Why is it necessary to check to see if this was a simulated device? In case someone created a real device and then used the simulator with the same device name? That'd be a very rare edge case we don't need to support.

I think we need a CLI option that says "don't touch the shadow", even if this is the 1st time creating this device and it's not in IoT yet at all. That would leave a minimal shadow you showed me in Slack, which I think is ok, could be a use case for DIY additions later:

```
{
  desired: {
    pairing: { state: 'paired', topics: [Object] },
    nrfcloud_mqtt_topic_prefix: 'dev/c4e08efa-0a95-4757-8be8-450e4138ae47/'
  },
  reported: { connection: { status: 'connected' } },
  delta: {
    pairing: { state: 'paired', topics: [Object] },
    nrfcloud_mqtt_topic_prefix: 'dev/c4e08efa-0a95-4757-8be8-450e4138ae47/'
  },
  metadata: {
    desired: { pairing: [Object], nrfcloud_mqtt_topic_prefix: [Object] },
    reported: { connection: [Object] }
  },
  version: 5
}
```

If that "don't touch" option is _not_ specified, then it does what the `-t` says, IIUC:

- if no `-t` specified (or `-t atv2`), create an atv2 shadow, overwriting whatever is already there, leaving whatever it doesn't overwrite.
- if `-t mss`, create an mss shadow....
- if `-t {blah}`, create a shadow with the blah stuff, as well as the minimal shadow above.
- if `-t myshadow.json`, create a shadow with the file's stuff, as well as the minimal shadow above.

Anything else?

Reply:
That's better. Forces the user to consider what kind of simulator they want, up front. 
What about `-t` by itself? Make that invalid? I like that better than `-t == -t atv2` like we have now, again forces the user to consider.